src/components/widgets/accounts-assets-liability-summary/accounts-assets-liability-summary.less

```diff
@@ -1,15 +1,15 @@
 .analytics .widget-item .content.accounts-assets-liability-summary {
+  height: 300px;
```

Comment:
@an-pavlovych again, please be careful with that: the widgets' height must not be changed or the all grid system will be broken.
I'll correct it for this PR.